pkgs/development/python-modules/google-auth/default.nix

```diff
@@ -85,6 +86,12 @@ buildPythonPackage rec {
     "--deselect=tests/transport/test__mtls_helper.py::TestDecryptPrivateKey::test_success"
   ];
 
+  disabledTestPaths = [
+    "samples/"
+    "system_tests/" # requires downloaded cloud SDK
+    "tests_async/transport/test_aiohttp_requests.py" # requires running event loop
```

Comment:
What's the error message?

Reply:
Added.

I'm having a rough time with the rebase onto the common merge base -- there's a lot of unrelated noise.